ddl/ddl_db_change_test.go

```diff
@@ -501,3 +499,56 @@ func (s *testStateChangeSuite) TestParallelDDL(c *C) {
 	callback = &ddl.TestDDLCallback{}
 	d.SetHook(callback)
 }
+
+func (s *testStateChangeSuite) testParrallelExecSQL(c *C, sql string) {
+	se, err := session.CreateSession(s.store)
+	_, err = se.Execute(context.Background(), "use test_db_state")
+	c.Assert(err, IsNil)
+
+	se1, err1 := session.CreateSession(s.store)
+	_, err = se1.Execute(context.Background(), "use test_db_state")
+	c.Assert(err1, IsNil)
+
+	var err2, err3 error
+	wg := sync.WaitGroup{}
+
+	callback := &ddl.TestDDLCallback{}
+	once := sync.Once{}
+	callback.OnJobUpdatedExported = func(job *model.Job) {
+		// sleep a while, let other job enque.
```

Comment:
How can we confirm that the other job is in the queue?

Reply:
Before this PR, the added test failed stably.

Reply:
But this test code is indeed unstable.